### DISCUSSION TITLE
Redis bug and stop container in 1 day when run in docker

### DISCUSSION THREAD

**Discussion by tidusvn05** [outline/outline#4670] [Self Hosting]
i got problem when use docker

----
1:S 11 Jan 2023 06:26:26.032 * Connecting to MASTER 185.246.90.205:8886
1:S 11 Jan 2023 06:26:26.032 * MASTER <-> REPLICA sync started
1:S 11 Jan 2023 06:26:26.301 # Error condition on socket for SYNC: Connection refused
1:S 11 Jan 2023 06:26:27.035 * Connecting to MASTER 185.246.90.205:8886
1:S 11 Jan 2023 06:26:27.036 * MASTER <-> REPLICA sync started
1:S 11 Jan 2023 06:26:27.353 # Error condition on socket for SYNC: Connection refused
1:S 11 Jan 2023 06:26:28.037 - DB 0: 27 keys (0 volatile) in 64 slots HT.
1:S 11 Jan 2023 06:26:28.037 . 7 clients connected (0 replicas), 2620016 bytes in use
1:S 11 Jan 2023 06:26:28.037 * Connecting to MASTER 185.246.90.205:8886
1:S 11 Jan 2023 06:26:28.038 * MASTER <-> REPLICA sync started
1:S 11 Jan 2023 06:26:28.324 # Error condition on socket for SYNC: Connection refused
1:S 11 Jan 2023 06:26:29.039 * Connecting to MASTER 185.246.90.205:8886
1:S 11 Jan 2023 06:26:29.039 * MASTER <-> REPLICA sync started
1:S 11 Jan 2023 06:26:29.312 # Error condition on socket for SYNC: Connection refused
1:S 11 Jan 2023 06:26:30.041 * Connecting to MASTER 185.246.90.205:8886
1:S 11 Jan 2023 06:26:30.041 * MASTER <-> REPLICA sync started
1:S 11 Jan 2023 06:26:30.313 # Error condition on socket for SYNC: Connection refused
1:S 11 Jan 2023 06:26:31.043 * Connecting to MASTER 185.246.90.205:8886
1:S 11 Jan 2023 06:26:31.043 * MASTER <-> REPLICA sync started
1:S 11 Jan 2023 06:26:31.318 # Error condition on socket for SYNC: Connection refused
1:S 11 Jan 2023 06:26:32.044 * Connecting to MASTER 185.246.90.205:8886
1:S 11 Jan 2023 06:26:32.044 * MASTER <-> REPLICA sync started
1:S 11 Jan 2023 06:26:32.320 # Error condition on socket for SYNC: Connection refused
1:S 11 Jan 2023 06:26:33.046 - DB 0: 27 keys (0 volatile) in 64 slots HT.
1:S 11 Jan 2023 06:26:33.046 . 7 clients connected (0 replicas), 2620016 bytes in use
1:S 11 Jan 2023 06:26:33.046 * Connecting to MASTER 185.246.90.205:8886
1:S 11 Jan 2023 06:26:33.046 * MASTER <-> REPLICA sync started
1:S 11 Jan 2023 06:26:33.320 # Error condition on socket for SYNC: Connection refused
1:S 11 Jan 2023 06:26:34.049 * Connecting to MASTER 185.246.90.205:8886
1:S 11 Jan 2023 06:26:34.049 * MASTER <-> REPLICA sync started
1:S 11 Jan 2023 06:26:34.333 # Error condition on socket for SYNC: Connection refused
````

only 1 user use it, but redis may heavy, and container down around 1-2 days
to continue use, i need restart containers: outline, redis, postgres


please help me, any solution.
thank you so much!

**Comment by tommoor** [comment]
This seems like a question for a redis community, I'm not sure you'll find much help configuring Redis here I'm afraid.
